src/Serilog.Sinks.Datadog.Logs/Serilog.Sinks.Datadog.Logs.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard1.3;net45;net461;net472</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard1.3;net45;net461;net472;net5.0;net6.0</TargetFrameworks>
```

Comment:
What is the reason to add `net6.0` as `net5.0` seems to enough?

I am sorry for my late comment.